ui/src/tracks/chrome_slices/index.ts

```diff
@@ -56,6 +56,18 @@ export interface Data extends TrackData {
   cpuTimeRatio?: Float64Array;
 }
 
+// Like a Slice, but only of the instantaneous kind and
+// only enough information to render it in the track
+export interface Instant {
```

Comment:
nit: If I'm not mistaken, only `title`, `tmAscent` and `color` are used in rendering an instantaneous track. Perhaps you were thinking that the additional fields might come in handy in the future if we change how we draw the diamond, but it would be just as easy to extend this type at that time, no? This isn't formal API, where you need to plan ahead.